apps/frontend/src/shared/i18n/locales/zh/worktrees.json

```
@@ -0,0 +1,61 @@
+{
+  "title": "工作树",
+  "description": "管理 Auto Claude 任务的隔离工作空间",
+  "selectProject": "选择一个项目以查看工作树",
+  "refresh": "刷新",
+  "error": "错误",
+  "noWorktrees": "没有工作树",
+  "noWorktreesDescription": "工作树在 Auto Claude 构建功能时自动创建。",
```

Biome (2.1.2)

[error] The key error was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)